tools/reviewer/web/src/app/dashboard/reviews-panel/diff/code-block/add-comment-buttons/add-comment-buttons.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="add-comment-buttons">
+  <div class="add-comment-button
+    {{line.isOpen?'':'closed-comment'}}
```

Comment:
try to keep a space for better readability like
`{{ line.isOpen ? '' : 'closed-comment' }}`

Reply:
and everywhere else as well.